src/includes/_judgment_text.scss

```diff
@@ -234,6 +268,18 @@
     margin-bottom: 0;
   }
 
+  &__hidden {
+    /* Visible to screenreaders */
+    position: absolute;
+    top: auto;
+    left: -10000px;
+
+    overflow: hidden;
+
+    width: 1px;
```

Comment:
I assume width: 0 is unacceptable?

Reply:
yeah, screenreaders won´t read it if it's 0 :upside_down_face: